x-pack/plugins/cloud/server/plugin.ts

```diff
@@ -14,6 +14,7 @@ import { registerCloudUsageCollector } from './collectors';
 import { getIsCloudEnabled } from '../common/is_cloud_enabled';
 import { parseDeploymentIdFromDeploymentUrl } from '../common/parse_deployment_id_from_deployment_url';
 import { decodeCloudId, DecodedCloudId } from '../common/decode_cloud_id';
+import { getFullCloudUrl } from '../common/utils';
 import { readInstanceSizeMb } from './env';
 
 interface PluginsSetup {
```

Comment:
__question:__ sorry for the post-review comment, but can we also update `mocks.ts` on the server-side to include mocks for Cloud plugin's `setup` and `start` contracts so that consumers can properly test the interaction with the Cloud plugin?